MANIFEST.in

```diff
@@ -1,8 +1,9 @@
-include *.in
-include *.py
+prune .github
 include *.md
```

Comment:
For the changelog?

Does the readme (RST) automatically get included?

Reply:
That's for `releasing.md`. The readme is apparently automatically included according to https://packaging.python.org/en/latest/guides/using-manifest-in/.